src/fastoad_cs25/models/geometry/geom_components/ht/components/compute_ht_mac.py

```diff
@@ -70,17 +70,17 @@ def compute(self, inputs, outputs):
         sweep_25_ht = inputs["data:geometry:horizontal_tail:sweep_25"]
         b_h = inputs["data:geometry:horizontal_tail:span"]
 
-        tmp = (
-            root_chord * 0.25 + b_h / 2 * math.tan(sweep_25_ht / 180.0 * math.pi) - tip_chord * 0.25
-        )
-
         mac_ht = (
             (root_chord**2 + root_chord * tip_chord + tip_chord**2)
             / (tip_chord + root_chord)
             * 2
             / 3
         )
-        x0_ht = (tmp * (root_chord + 2 * tip_chord)) / (3 * (root_chord + tip_chord))
+
+        x0_ht = (root_chord / 4) + (b_h / 6) * math.tan(sweep_25_ht / 180.0 * math.pi) * (
```

Comment:
Since we do modifications here, can we take the opportunity to put "rad" as input unit and get rid of this conversion?

Reply:
Corrected in [a81f76f](https://github.com/fast-aircraft-design/FAST-OAD_CS25/pull/45/commits/a81f76ffb5eedb442f8fc08a514ad1d8795daf43)